xbmc/windowing/gbm/OptionalsReg.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ *      Copyright (C) 2005-2017 Team XBMC
+ *      http://xbmc.org
```

Comment:
Maybe update the url in both files to all kodi.tv

Reply:
done